spec/spec_helper.rb

```diff
@@ -1,5 +1,44 @@
 require 'simplecov'
 require 'simplecov-console'
+require '/Users/saritaradia/Desktop/Projects/chitter-challenge/chitter-challenge/spec/database_setup.rb'
+require '/Users/saritaradia/Desktop/Projects/chitter-challenge/chitter-challenge/app.rb'
+require '/Users/saritaradia/Desktop/Projects/chitter-challenge/chitter-challenge/spec/Support/database_cleaner.rb'
```

Comment:
I'm not 100% sure, but these file paths look like they might only work on your local system - if someone were to fork this, you might need more general requires or something like require_relative './spec/database_setup.rb' ?